config/config.go

```diff
@@ -409,6 +410,12 @@ func New(logger *zap.SugaredLogger, osArgs []string) (config, error) {
 		return cfg, err
 	}
 
+	mainFS.DurationVar(&cfg.Controller.DisruptionDeletionTimeout, "disruption-deletion-timeout", time.Minute*(-1), "If the deletion time of the disruption is greater than the delete timeout, the disruption is marked as stuck on removal")
```

Comment:
Why a negative minute? We seem to assume it's a positive value in the code (and in the real value we configure)

Reply:
I assume this is meant to disable this feature if not set, but I think it should always be on, and I don't think we'd handle a negative value correctly

Reply:
Copy, Past error 🙈
I can add a default value to 15 minutes. WDYT?